src/components/Card/Card.js

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Card.propTypes = {
+  image: PropTypes.node.isRequired,
+  title: PropTypes.string,
+  text: PropTypes.string,
+}
+
+export default function Card({ text, title, image }) {
+  return (
+    <WrapperList>
+      <li>
+        <img src={image} alt="" />
+        <h2>{title}</h2>
+        <p>{text}</p>
+      </li>
+    </WrapperList>
+  )
+}
+
+const WrapperList = styled.ul`
+  background-image: linear-gradient(
+    to top,
+    #0c3483 0%,
+    #a2b6df 100%,
+    #6b8cce 100%,
+    #a2b6df 100%
```

Comment:
if you are using the same colors more often maybe you could set variables for the colors